qipr_approver/approver/api/tags.py

```diff
@@ -0,0 +1,30 @@
+from django.http import JsonResponse, HttpResponse, QueryDict
+from approver.utils import get_model_from_string
+
+def tags(request):
+    if request.method == 'POST':
```

Comment:
You could make this a GET instead of a post and use url parameters. Probably best practice, not really important however